WTWSMS/history/titles/b_ostia.txt

```diff
@@ -517,7 +517,7 @@
 	holder = 0
 }
 408.1.1 = {
-	holder = 1006011 # Auchenius Bassus
+	holder = 1006011 # Auchenius Bassus (II)
```

Comment:
Wasn't he the first consul named Auchenius Bassus?

Reply:
Yes, he first consul named Auchenius Bassus, but second Auchenius Bassus in Anicii family.

Reply:
Put that in the character file then, or expand the comment.

Reply:
Auchenius Bassus I was not consul , but his son Auchenius Bassus II and grandson Auchenius Bassus III were consuls.

Reply:
I think the original numbers are better, someone can confuse the numbers you suggest with regnal numbers.

Reply:
@Symmachus 
I and @loup99 discussed that and the convention is to use the numbers as regnal numbers, so if 1006011 is the first Auchenius Bassus to rule b_ostia, he should be "I" in the comment, not "II". You can add additional info in the comment though.